src/MessageContainer.js

```diff
@@ -109,13 +117,15 @@ export default class MessageContainer extends React.Component {
       message.user = {};
     }
 
+    const position = messagePosition(message, this.props.user);
```

Comment:
You don't need a variable here. You can use `messagePosition` directly in  `messageProps`.